index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Home</title>
+</head>
+
+<body>
+    <section class="home-section">
+        <header class="header1">
+            <div class="header1-content">
+                <ul>
+                   <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+        </header>
+        <header>
+            <img src="img/menu.png" width="40" height="40" alt="hamburger-button" class="hamburger-button">
+            <img src="img/close.png" width="40" height="40" alt="close-btn" class="close-btn">
+            <nav class="navbar" id="navbar-list">
+                <ul>
+                    <li id="logo"><a href="index.html" rel="noreferrer noopener"><img src="img/logo.jpg" alt="logoimage" width="130" height="50"></a></li>
+                    <li><a href="about.html" rel="noreferrer noopener">About</a></li>
+                    <li><a href="#" rel="noreferrer noopener">Program</a></li>
+                    <li><a href="#" rel="noreferrer noopener">Join</a></li>
+                    <li><a href="#" rel="noreferrer noopener">Sponsor</a></li>
+                    <li><a href="#" rel="noreferrer noopener">News</a></li>
+                    <li id="websummit"><a href="#" rel="noreferrer noopener">Web Summit 2021</a></li>
+                </ul>
+            </nav>
+        </header>
+        <div class="main-title">
+            <h2>"Hello,Tech World"</h2>
```

Comment:
- [ ] OPTIONAL
To ensure proper use of HTML semantics kindly ensure that there's at least 1, but no more than 1, `h1` tag on each page.